port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example

```diff
@@ -1,2 +1,6 @@
-OCEAN__PORT__CLIENT_ID="<port-client-id>"
-OCEAN__PORT__CLIENT_SECRET="<port-client-secret>"
+OCEAN__PORT__CLIENT_ID={{ cookiecutter.port_client_id }}
+OCEAN__PORT__CLIENT_SECRET={{ cookiecutter.port_client_secret }}
+OCEAN__INTEGRATION__IDENTIFIER={{ cookiecutter.integration_slug }}
+OCEAN__PORT__BASE_URL={% if cookiecutter.is_us_region %}https://api.us.getport.io{% else %}https://api.getport.io{% endif %}
+OCEAN__EVENT_LISTENER__TYPE=POLLING
```

Comment:
```suggestion
OCEAN__EVENT_LISTENER={"type": "POLLING"}
```

Reply:
why do you think we need this changed? if i change it, the .env file generated will be different

Reply:
decided to leave as is